src/test/java/io/github/christophermanahan/carnitas/HTTPServerTest.java

```diff
@@ -20,23 +22,27 @@
     @BeforeEach
     void setup() {
         sent = new ArrayList<>();
+        request = "GET http://localhost:80/simple_get HTTP/1.1";
         logger = new TestLogger();
     }
 
     @Test
-    void sendsHTTPResponseAfterReceivingData() {
-        connection = new TestConnection("GET simple_get HTTP/1.1", sent);
+    void sendsHTTPResponsesWhileReceivingData() {
```

Comment:
I think I'm confused what this is testing. Perhaps Damon has you going in a different direction, but I was expecting there to be something that connections to `HTTPServer`, it sends a request, and then receives a response.

Reply:
Nevermind, I understand what's going on now. I think I was just confused with how things were setup. I think this: https://github.com/christophermanahan/carnitas/pull/7#discussion_r244834518
might clarify things